src/main/java/org/opensearch/jobscheduler/transport/action/TransportGetScheduledInfoAction.java

```diff
@@ -192,37 +191,34 @@ protected GetScheduledInfoNodeResponse nodeOperation(GetScheduledInfoNodeRequest
                                         );
                                         scheduleMap.put("interval", intervalSchedule.getInterval());
                                         scheduleMap.put("unit", intervalSchedule.getUnit().toString());
+                                        scheduleMap.put(
+                                            "delay",
+                                            jobInfo.getJobParameter().getSchedule().getDelay() != null
+                                                ? jobInfo.getJobParameter().getSchedule().getDelay()
+                                                : "none"
+                                        );
                                     } else if (jobInfo.getJobParameter().getSchedule() instanceof CronSchedule cronSchedule) {
                                         scheduleMap.put("type", CronSchedule.CRON_FIELD);
                                         scheduleMap.put("expression", cronSchedule.getCronExpression());
                                         scheduleMap.put("timezone", cronSchedule.getTimeZone().getId());
+                                        scheduleMap.put(
+                                            "delay",
+                                            jobInfo.getJobParameter().getSchedule().getDelay() != null
+                                                ? jobInfo.getJobParameter().getSchedule().getDelay()
+                                                : "none"
+                                        );
                                     } else {
                                         scheduleMap.put("type", "unknown");
                                     }
 
                                     jobDetails.put("schedule", scheduleMap);
-
-                                    // Add delay information
-                                    jobDetails.put(
-                                        "delay",
-                                        jobInfo.getJobParameter().getSchedule().getDelay() != null
-                                            ? jobInfo.getJobParameter().getSchedule().getDelay()
-                                            : "none"
-                                    );
                                 }
 
                                 // Add jitter and lock duration
                                 jobDetails.put(
                                     "jitter",
                                     jobInfo.getJobParameter().getJitter() != null ? jobInfo.getJobParameter().getJitter() : "none"
                                 );
-                                jobDetails.put(
```

Comment:
Let's keep `lock_duration` in. At this level, `lock_duration` represents the maximum amount of time a lock will be held for after its acquired. If a job runs and fails to release a lock in time (for example if the job fails while running), then JS has a mechanism to release the lock so that its not help in perpetuity.